rmf_task/src/rmf_task/requests/ChargeBattery.cpp

```diff
@@ -92,6 +92,10 @@ rmf_task::ConstRequestPtr ChargeBattery::make(
   charge_battery->_pimpl->drain_battery = drain_battery;
   charge_battery->_pimpl->invariant_duration =
     rmf_traffic::time::from_seconds(0.0);
+
+  // use charge start time to construct an ID: TODO: better way
+  auto time_id = start_time.time_since_epoch().count();
```

Comment:
We already append a UUID to `_pimpl->id` in line 80. Hence this is not required.

Reply:
Forgot to remove. Removed.